dashboard/app/assets/javascripts/client_state.js

```diff
@@ -25,6 +25,14 @@ dashboard.clientState = {};
  */
 dashboard.clientState.EXPIRY_DAYS = 365;
 
+/**
+ * Maximum number of lines of code that can be stored in the cookie
+ * @type {number}
+ * @private
+ */
+dashboard.clientState.MAX_LINES_TO_SAVE = 1000;
```

Comment:
Is this an internal implementation detail?  You could just make it a variable inside the closure.